sound/soc/codecs/rt1320-sdw.c

```diff
@@ -535,6 +535,9 @@ static int rt1320_read_prop(struct sdw_slave *slave)
 	/* set the timeout values */
 	prop->clk_stop_timeout = 64;
 
+	/* BIOS may set wake_capable for Windows. But wake_capable is not needed for Linux */
```

Comment:
@bardliao I dont know if we should mention Windows at all. Can we simply state that the BIOS may set it as wake-capable and leave it at that?

Reply:
@ranj063 @ujfalusi done

Reply:
@bardliao its still there in the comment in the code?